doc/classes/ParallaxLayer.xml

```diff
@@ -60,10 +61,13 @@
 	</methods>
 	<members>
 		<member name="motion_mirroring" type="Vector2" setter="set_mirroring" getter="get_mirroring">
+			 Mirroring of the [code]ParallaxLayer[/code], reapeat the [code]ParallaxLayer[/code]. Useful to create an infinite scrolling background. If an axis is set to 0 then that axis will have no mirroring. 
 		</member>
 		<member name="motion_offset" type="Vector2" setter="set_motion_offset" getter="get_motion_offset">
+			 Start offset of the [code]ParallaxLayer[/code].
```

Comment:
The ParallaxLayer's offset relative to the parent ParallaxBackground's [member ParallaxBackground.scroll_offset].